spec/features/events_spec.rb

```diff
@@ -166,6 +167,42 @@
     end
   end
 
+  scenario "logged in as Organizer I can send acceptance and then rejection emails and by that change the status notification flag" do
+    login(:organizer)
+    event = FactoryGirl.create(:event_with_accepted_applications, :in_selection_phase)
+
+    event.application_letters.each do |application|
```

Comment:
The default value is already `false`, this block is only here to make sure this test has still a meaning in case the factory's default value switches to true